src/UploadedFile.php

```diff
@@ -277,11 +216,6 @@ public function moveTo($targetPath)
         }
     }
 
-    /**
-     * {@inheritdoc}
-     *
-     * @return int|null The file size in bytes or null if unknown.
```

Comment:
The additional phpdocs in the methods below like `getError` seem useless as well as they are just copy pasted from the interface